src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/AbstractJiraCoreScenario.kt

```diff
@@ -26,7 +30,8 @@ abstract class AbstractJiraCoreScenario {
         val metrics = mutableListOf<ActionMetric>()
         val actionMeter = ActionMeter.Builder(
             output = CollectionActionMetricOutput(metrics)
-        ).build()
+        ).appendPostMetricHook(DrillDownHook(JavascriptW3cPerformanceTimeline(driver as JavascriptExecutor)))
```

Comment:
The opt-in is roundabout, isn't it? That's because historically we introduced a *specific* API for drilldown (`W3cPerformanceTimeline`) and only later we *generalized* it to `PostMetricHook`. Now the `DrillDownHook` acts as a bridge.

We can make it simpler if we make `JavascriptW3cPerformanceTimeline` implement the `PostMetricHook` directly.
We couldn't do that with `W3cPerformanceTimeline` due to SPI compatibility, but we can do that with the impl, because it's `final`. Then we could do this:
```suggestion
        ).appendPostMetricHook(JavascriptW3cPerformanceTimeline(driver as JavascriptExecutor))
```

Reply:
Makes sense :+1:  :D